dtfs-central-api/api-tests/v1/utilisation-reports/post-keying-data.api-test.ts

```diff
@@ -342,15 +342,17 @@ describe(`POST ${BASE_URL}`, () => {
           ...aTfmFacilityAmendment(),
           value: 350000,
           status: AMENDMENT_STATUS.COMPLETED,
-          effectiveDate: dateAfterReportPeriodEnd.getTime(),
+          // Effective dates are stored in unix epoch time in seconds not milliseconds.
+          effectiveDate: Math.round(dateAfterReportPeriodEnd.getTime() / 1000),
```

Comment:
Could we create a helper for `Math.round( ... / 1000)`? Repeated 11x times. Maybe in "common"?